deny.toml

```diff
@@ -4,7 +4,7 @@
 #   id="RUSTSEC-2021-0139",
 #   reason="The maintainer has advised that this crate is deprecated and will not receive any maintenance. https://rustsec.org/advisories/RUSTSEC-2021-0139.html"
 #  },
-ignore = []
+ignore = ["instant@0.1.13"]
```

Comment:
I had to add this. Not sure it wasn't flagging before, since this library was moved to "unmaintained" 6 months ago. But it's still a dependency in ethers...which has itself [moved to unmaintained status](https://github.com/gakonst/ethers-rs) as of a few weeks ago.

Reply:
yeah we have to update ethers to alloy anyway, #1111, it should be ok to ignore for now

Reply:
OK. Fixed